cloudbuild.yaml

```diff
@@ -3,17 +3,62 @@
 
 steps:
 # Perform maven build, omitting local docker operations
-- name: 'maven:3.3.9-jdk-8'
-  args: ['mvn', '-B', '-P-local-docker-build', '-P-test.local', '-Ddocker.tag.long=$_DOCKER_TAG', 'clean', 'install']
+- name: 'gcr.io/cloud-builders/java/mvn:3.5.0-jdk-8'
+  args:
+    - '--batch-mode'
+    - '-P-local-docker-build'
+    - '-P-test.local'
+    - '-Ddocker.tag.long=${_DOCKER_TAG}'
```

Comment:
Yes, as maven only download the Jetty version but does not generate any tag for the docker image it will not be disruptive for cloudbuilder to tag and push the image.